docs/icd.md

```diff
@@ -8,59 +8,88 @@
 
 ## Overview
 
-This document details the common software interfaces required by all services in the Arrow ecosystem.
+This document details the endpoints provided by the `svc-pricing` service.
+
+Currently, the server is configured to be live on `localhost:50051`, and is subject to change when deployed to production. This address can be interfaced by any API platforms like Postman, or even `curl`. 
+
+However, `svc-pricing` is intended to be consumed by other Arrow micro-services through the `svc-pricing-client` library, rather than the server being directly connected to and interacted with frontend clients.
 
-Each service may add additional interfaces.
 
 Attribute | Description
 --- | ---
-Status | Draft
+Status | :yellow_circle: Development
 
 ## Related Documents
 
 Document | Description
 --- | ---
-:construction: Requirements & User Stories :construction: | Requirements and user stories
+[Pricing Model](https://docs.google.com/spreadsheets/d/1mjPtaIn3E5m7r4nyKt_sJKG9BSFm2ty7Gzo7OqERxwo) | Unit economics and pricing mechanism of flights. The core logic of `svc-pricing` is largely derived from the pricing model.
```

Comment:
link the High-Level ICD when it's merged

Reply:
[2baaed0](https://github.com/Arrow-air/svc-pricing/pull/2/commits/2baaed01064066accaa5ec859762f38787027c01)

GH actions to be re-run when that high-level ICD PR is merged.